client/me/reauth-required/index.jsx

```diff
@@ -54,7 +54,7 @@ module.exports = React.createClass( {
 			);
 		} else {
 			codeMessage = this.translate(
-				'Please enter the verification code generated by your Authenticator mobile application.'
+				'Please enter the verification code generated by your authenticator app.'
```

Comment:
:disappointed: 26 existing translations will be lost with this change.

Reply:
I'll look if we can manage to keep them despite this message :)